### ISSUE TITLE
Hyperlink needs more documentation

### ISSUE THREAD

**Issue by csharpfritz** [FritzAndFriends/BlazorWebFormsComponents#225]
Needs some blazor implementation details added

**Comment by hishamco** [comment]
What kind of docs that we need here?

**Comment by akoranteng** [comment]
Can you share my information regarding what you mean by Hyperlink needs more documentation. What hyperlinks are you referring to here. Please elaborate

**Comment by csharpfritz** [comment]
Yes... the documents in the docs folder need to be updated to include
information about what is supported by the current component and what needs
to be done.


Jeff

**Comment by akoranteng** [comment]
Any updates on when this will be updated. Just a follow up.
